src/pkg/ldap/ldap.go

```diff
@@ -179,7 +179,30 @@ func (s *Session) SearchUser(username string) ([]model.User, error) {
 			}
 			u.GroupDNList = groupDNList
 		}
+
 		u.DN = ldapEntry.DN
+		// Search for nested group memberships (if AdminFilter is set)
+		if s.groupCfg.AdminFilter != "" {
+			nestedFilter, err := createUserSearchFilter(s.groupCfg.AdminFilter, s.basicCfg.UID, username)
```

Comment:
Because there are some existing ldap group settings, the admin user search filter should apply these settings too.

1. LDAP Group Base DN
2. LDAP Group Filter
3. LDAP Group Search Scope